Gemfile

```diff
@@ -27,6 +27,7 @@ gem 'everypolitician-pull_request', github: 'everypolitician/everypolitician-pul
 gem 'everypolitician-dataview-terms', github: 'everypolitician/everypolitician-dataview-terms'
 gem 'field_serializer', github: 'everypolitician/field_serializer'
 gem 'csvlint'
+gem 'deep_merge'
```

Comment:
I was a little surprised that `rubocop` pick up on this, but it seems we've never actually turned it on for this repo (presumably because of how many errors we currently have!)

This isn't a particularly serious case, so I think we can live with it for now, but it would probably be worth trying to turn rubocop on fairly soon, even with a fairly large TODO file, so we at least start moving in the right direction.